fixtures/tests/_fixtures/test_monkeypatch.py

```diff
@@ -310,16 +310,17 @@ def test_patch_boundmethod_with_boundmethod(self):
         with fixture:
             INST_C.foo()
         self.assertEqual(oldmethod, INST_C.foo)
+        sentinel = object()
+        self.assertEqual(sentinel, INST_C.__dict__.get('foo', sentinel))
 
     def test_patch_unboundmethod_with_staticmethod(self):
         oldmethod = C.foo
         fixture = MonkeyPatch(
             'fixtures.tests._fixtures.test_monkeypatch.C.foo',
             D.bar_static_args)
         with fixture:
-            c = C() 
-            tgtslf, arg = c.foo(1)
-            self.expectThat(tgtslf, Is(c))
+            tgtslf, arg = INST_C.foo(1)
```

Comment:
Was this change intentional? The class is being patched and the test checks against an instance of that class which was instantiated before patching. But the way this is done provides no check that INST_C.foo is actually patched and not the original method.